CONTRIBUTORS.rst

```diff
@@ -0,0 +1,10 @@
+Contributors
+============
+
+* Alexis Metaireau <alexis@mozilla.com>
+* Fil <fil@rezo.net>
+* Mathieu Agopian <mathieu@agopian.info>
+* Mathieu Leplatre <mathieu@mozilla.com>
+* Nicolas Perriault <nperriault@mozilla.com>
+* Rémy Hubscher <rhubscher@mozilla.com>
+* Tarek Ziade <tarek@mozilla.com>
```

Comment:
Cool, we need to make sure we update this file each time we do a release (if we had templates of what we should do for a release, we could include this step)